src/Microsoft.AspNetCore.Mvc.Core/Controllers/ControllerFeature.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Mvc.Controllers
+{
+    /// <summary>
+    /// The list of controllers types in an MVC application.
```

Comment:
This should probably reference the part manager or application parts in some way